### ISSUE TITLE
The package is of bad quality

### ISSUE THREAD

**Issue by GoogleCodeExporter** [niteshmourya/app-inventor-for-android#1885]
```
// Steps to reproduce the problem.

 Clicked on Debian Download link(Chromium in Ubuntu 11.04). Software center opened, clicked install. Starts installing then presented with this message:

// What happened.  (Don't just say: "It didn't work.")

"The installation of a package which violates the quality standards isn't 
allowed. This could cause serious problems on your computer. Please contact the 
person or organisation who provided this package file and include the details 
beneath."

Details:
"Lintian check results for 
/home/[myUsername]/Downloads/appinventor-setup_1.1_all (1).deb:
E: appinventor-setup: arch-independent-package-contains-binary-or-object 
usr/google/appinventor/commands-for-Appinventor/adb
E: appinventor-setup: arch-independent-package-contains-binary-or-object 
usr/google/appinventor/commands-for-Appinventor/emulator
E: appinventor-setup: arch-independent-package-contains-binary-or-object 
usr/google/appinventor/commands-for-Appinventor/mksdcard
"

// What you think the correct behavior should be.

I think it should not have a warning and the option should be to run the app 
instead of re-installing it.

/*Don't forget to mention which device you have, and which version of Android
is installed on it.  (Find it under Home > Menu > Settings > About
phone.)  Please also tell us what Operating System your computer is running
and what browser you were using to access App Inventor.*/

I have a motoroal Atrix 4G, AT&T not rooted(yet). System version 4.1.83. 
Running Natty Narwhal Ubuntu.



TL;DR
The install package came up with an error.



```

Original issue reported on code.google.com by `Scotty....@gmail.com` on 3 Oct 2011 at 12:17
